docs/documentation/indexing/create_index.mdx

```diff
@@ -246,6 +247,23 @@ type `TEXT[]` can be passed into `text_fields`, and columns of type `INT[]` can
   supported.
 </Note>
 
+### Range Fields
+
+Columns of type `int4range`, `int8range`, `numrange`, `tsrange`, and `tstzrange` can be indexed as `range_fields`.
+
+```sql
+CALL paradedb.create_bm25(
+  index_name => 'search_idx',
+  table_name => 'mock_items',
+  key_field => 'id',
+  range_fields => paradedb.field('weight_range')
+);
+```
+
+<ParamField body="stored" default={true}>
+  Whether the original value of the field is stored.
+</ParamField>
+
```

Comment:
It might be helpful to link to your page for `range_term` from here, so curious users can easily see some examples.